davidpak/SmartNote
# Added simple tests for file validity

## What type of PR is this? (check all applicable)


## Description
Added very simple tests to ensure that the vector databases and embeddings can be properly created and utilized with various inputs.

## QA Instructions, Screenshots, Recordings
Ensure that `pytest` is installed on your local machine by running `pip install pytest`. Each test can be ran individually as of right now. Will add collective testing in next push.

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?
Grab input from users and test LLM generation.
